app/javascript/controllers/status_controller.js

```diff
@@ -9,17 +9,40 @@ export default class extends Controller {
     }
   }
 
+  disconnect() {
+    this.stopRefreshing()
+  }
+
   startRefreshing() {
-    setInterval(() => {
+    this.interval = setInterval(() => {
       this.load()
     }, this.data.get("refreshInterval"))
   }
 
+  stopRefreshing() {
+    if(this.interval) {
+      clearInterval(this.interval)
+    }
+  }
+
   load() {
     fetch(this.data.get("url"))
-    .then(response => response.text())
-    .then(html => {
-      this.element.innerHTML = html
-    })
+      .then(response => response.text())
+      .then(string => JSON.parse(string))
+      .then(json => {
+        this.element.innerHTML = `
+          <p><strong>Promoted:</strong> ${json.promoted}</p>
```

Comment:
Any thoughts on HTML templates here? Injecting HTML in JS has always felt... odd.